website/components/Compare.module.css

```
@@ -1,48 +1,58 @@
   display: grid;
   grid-template-columns: 1.4fr 1.3fr 1fr 1fr 1fr;
   gap: 16px;
-  padding: 20px 24px;
-  border-bottom: 1px solid var(--charcoal);
+  padding: 16px 24px;
+  border: 1px solid var(--hairline);
+  border-bottom: none;
   align-items: center;
   font-size: 14px;
-  letter-spacing: 0.12px;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)